nrm_django/grants/models.py

```diff
@@ -178,11 +227,9 @@ class Grant(models.Model):
     cooperator_agreement_number = models.CharField(
         max_length=34, blank=True, null=True
     )  # Is this used to key to a cooperator agreement?
-    gid = models.CharField(max_length=16, blank=True, null=True, editable=False)
+    gid = models.CharField("Agreement ID", max_length=16, blank=True, null=True)
```

Comment:
I've seen this as "Agreement ID" in the application, but people seem to talk about it as an "Agreement Number". Is "Agreement No." a better label here?

Reply:
yes!